pkgs/build-support/make-darwin-bundle/default.nix

```diff
@@ -12,14 +12,14 @@
 
 writeShellScript "make-darwin-bundle-${name}" (''
   function makeDarwinBundlePhase() {
-    mkdir -p "$out/Applications/${name}.app/Contents/MacOS"
-    mkdir -p "$out/Applications/${name}.app/Contents/Resources"
+    mkdir -p "${!outputBin}/Applications/${name}.app/Contents/MacOS"
```

Comment:
you must escape outputBin

see https://grep.app/search?q=outputBin&filter[repo][0]=NixOS/nixpkgs&filter[lang][0]=Nix&filter[path][0]=pkgs/

Reply:
Ah, right. Thanks! 🤦